### PR TITLE
Urlscan.io util class for dns enumeration

### PR DESCRIPTION
**Pull request by rachejazz**
```
╰──> λ ./maryam -e dns_search -e urlscan -d owasp.org
[*] [URLSCAN] Searching in urlscan.io...
[*] HOSTNAMES
[*] 	brainbreak.owasp.org
[*] 	www.owasp.org
[*] 	calltobattle.owasp.org
[*] 	dev.owasp.org
[*] 	training.owasp.org
[*] 	cheatsheetseries.owasp.org
[*] 	wiki.owasp.org
```
And fixed some grammatical/indentations

### CODE REVIEW

File: core/util/urlscan.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        self.framework.verbose('[URLSCAN] Searching in urlscan.io...')
+        try:
+            req = self.framework.request(self.urlscan_api)
+            result_json = req.json()
+        except:
+            self.framework.error('[URLSCAN] Connection Error')
+            return
+        else:
+            if not result_json['total']:
+                self.framework.verbose('[URLSCAN] Not yet in our database. You can still submit a scan at https://urlscan.io/ \n\tand then retry scanning.')
```
Comment (saeeddhqan) [inline core/util/urlscan.py]: A simple message is better.

File: modules/search/stackoverflow.py [hunk 2]
```diff
@@ -93,7 +93,7 @@ def module_api(self):
 def module_run(self):
 	output = module_api(self)
 	for i in range(len(output['links'])):
-		self.output(f"{output['links'][i][1]}", 'G')
-		self.output(f"\t{output['links'][i][0]}", 'G')
+		self.output(f"{output['links'][i][1]} \n\t{output['links'][i][0]}", 'G')
+#		self.output(f"\t{output['links'][i][0]}", 'G')
```
Comment (saeeddhqan) [inline modules/search/stackoverflow.py]: remove comment.

**Review by saeeddhqan — COMMENTED** [review]
Add it to social_nets module as a source.

**Comment by rachejazz** [comment]
> Add it to social_nets module as a source.

done

**Pull request merged**